envs/conda_build_config.yaml

```diff
@@ -80,13 +82,17 @@ google_pasta:
   - 0.2
 googleapis_common_protos:
   - 1.52.*
+gmock:
```

Comment:
Do we need both `gmock` and `gtest`? I think they are the same thing these days.

Reply:
It looks like they do still build into separate libraries but they are uniformly versioned. We could just use a single pin (`gtest`) for both test and mock I think - at the expense of a little bit of readability, but it's more proper - these two should stay linked.

Reply:
Okay.